### PR TITLE
*: Change compatibility description for add index acceleration&PiTR

### PR DESCRIPTION
**Pull request by Benjamin2037**
### What is changed, added or deleted? (Required)

Add more description to make compatibility issue more clean between add index acceleration and PiTR. Let user can clearly know what will happen.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [X] master (the latest development version)
- [X] v6.4 (TiDB 6.4 versions)
- [X] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions) 
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -869,7 +869,7 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 >
 > 当前索引加速功能与 [PITR (Point-in-time recovery)](/br/point-in-time-recovery.md) 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：
 >
-> - 如果先启动 PITR 备份任务，再添加索引，此时即使索引加速功能打开，也不会使用加速索引功能，但不影响索引兼容性。
+> - 如果先启动 PITR 备份任务，再添加索引，此时即使索引加速功能打开，也不会使用加速索引功能，但不影响索引兼容性, 由于 PiTR 备份任务会一直运行，相当于索引加速功能被禁用，后续迭代会实现两个功能的兼容使用。
```
Comment (qiancai) [inline system-variables.md:872]:
```suggestion
> - 如果先启动 PITR 备份任务，再添加索引，此时即使索引加速功能打开，也不会使用加速索引功能，但不影响索引兼容性, 由于 PITR 备份任务会一直运行，相当于索引加速功能被关闭。
```

Comment (qiancai) [reply system-variables.md:872]: 为了避免后续版本计划发生变化，文档里一般尽量避免提到未来的计划和事项。

**Review by tangenta — APPROVED** [review]

**Comment by Oreoxmt** [comment]
@Benjamin2037 https://docs.pingcap.com/zh/tidb/dev/pitr-known-issues#%E7%B4%A2%E5%BC%95%E5%8A%A0%E9%80%9F%E5%8A%9F%E8%83%BD%E4%B8%8E-pitr-%E5%8A%9F%E8%83%BD%E4%B8%8D%E5%85%BC%E5%AE%B9 这里也需要修改

File: system-variables.md [hunk 2]
```diff
@@ -869,7 +869,7 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 >
 > 当前索引加速功能与 [PITR (Point-in-time recovery)](/br/point-in-time-recovery.md) 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：
 >
-> - 如果先启动 PITR 备份任务，再添加索引，此时即使索引加速功能打开，也不会使用加速索引功能，但不影响索引兼容性。
+> - 如果先启动 PITR 备份任务，再添加索引，此时即使索引加速功能打开，也不会使用加速索引功能，但不影响索引兼容性, 由于 PITR 备份任务会一直运行，相当于索引加速功能被关闭。
```
Comment (Oreoxmt) [inline system-variables.md:872]:
```suggestion
> - 如果先启动 PITR 备份任务，再添加索引，此时即使索引加速功能打开，也不会使用加速索引功能，但不影响索引兼容性。由于 PITR 备份任务会一直运行，相当于索引加速功能被关闭。
```

**Comment by Benjamin2037** [comment]
> @Benjamin2037 https://docs.pingcap.com/zh/tidb/dev/pitr-known-issues#%E7%B4%A2%E5%BC%95%E5%8A%A0%E9%80%9F%E5%8A%9F%E8%83%BD%E4%B8%8E-pitr-%E5%8A%9F%E8%83%BD%E4%B8%8D%E5%85%BC%E5%AE%B9 这里也需要修改

这里不需要修改了，PITR 这块说明不需要修改了。

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**